redis.conf

```diff
@@ -1449,6 +1449,13 @@ lua-time-limit 5000
 #
 # cluster-node-timeout 15000
 
+# The default cluster bus port parameter is 0. If this parameter is not set in the config file
+# then  cluster bus port = command port+ 10000; Otherwise in the cluster uou need  explicitly 
+# mention it in the command "cluster meet":  for example:  cluster meet ip port cluster-port
+#
+# Cluster port 
```

Comment:
```suggestion
# The cluster port is the port that the cluster bus will listen for inbound connections on. When set 
# to the default value, 0, it will be bound to the command port + 10000. Setting this value requires 
# you to you to specify the cluster bus port when executing cluster meet.
```

?